Configurations/unix-Makefile.tmpl

```diff
@@ -561,15 +562,14 @@ uninstall_docs: uninstall_man_docs uninstall_html_docs
 
 install_fips: install_sw
 	@$(ECHO) "*** Installing FIPS module configuration"
-	@$(ECHO) "fipsinstall $(DESTDIR)$(MODULESDIR)/$(FIPSMODULENAME).cnf"
-	@openssl fipsinstall -module $(DESTDIR)$(MODULESDIR)/$(FIPSMODULENAME) \
-		-out $(DESTDIR)$(MODULESDIR)/$(FIPSMODULENAME).cnf \
-		-macopt 'key:$(FIPSKEY)'
+	$(SRCDIR)/util/wrap.pl $(SRCDIR)/apps/openssl fipsinstall \
```

Comment:
`$(SRCDIR)/util/wrap.pl $(SRCDIR)/apps/openssl` should be `$(BLDDIR)/util/wrap.pl $(BLDDIR)/apps/openssl`

Reply:
Yeah, right. I forgot about the out-of-tree builds. Since the hunks of the last two commits collide, I integrated and cleaned up the fixups immediately and [force-pushed](https://github.com/openssl/openssl/compare/d87bc00811becea20e85015b91ed3faea158f810..e45552c442351cc6eec3b4515e1890a2c5378505) the result.